src/rust/engine/src/core.rs

```diff
@@ -166,8 +166,9 @@ pub struct Function(pub Key);
 impl Function {
   fn pretty_print(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
     let Function(key) = self;
+    let module = externs::project_str(&externs::val_for(&key), "__module__");
```

Comment:
John had wanted to also include the line number here. Do you think that's useful to add, or would that make the message too noisy?

Reply:
Adding in the next commit. Thanks!